### ISSUE TITLE
Support PyJWT >2

### ISSUE THREAD

**Issue by testworksau** [NiklasRosenstein/python-github-bot-api#5]
There's a conflict with PyGitHub >=1.55 and this library which depends on PyJWT<2.0.0.

It would be good if this library can be updated to use a newer version of PyJWT so that we can take the latest PyGitHub version.

```
ERROR: Cannot install <packagename> because these package versions have conflicting dependencies.

The conflict is caused by:
    pygithub 1.55 depends on pyjwt>=2.0
    github-bot-api 0.3.3 depends on PyJWT<2.0.0 and >=1.7.1
    pygithub 1.55 depends on pyjwt>=2.0
    github-bot-api 0.3.1 depends on PyJWT<2.0.0 and >=1.7.1
    pygithub 1.55 depends on pyjwt>=2.0
    github-bot-api 0.3.0 depends on PyJWT<2.0.0 and >=1.7.1
    pygithub 1.55 depends on pyjwt>=2.0
    github-bot-api 0.2.0 depends on PyJWT<2.0.0 and >=1.7.1
    pygithub 1.55 depends on pyjwt>=2.0
    github-bot-api 0.1.0 depends on PyJWT<2.0.0 and >=1.7.1

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
ERROR: 1
```

**Comment by NiklasRosenstein** [comment]
Hey @testworksau , sure thing. Will do this in a separate version after we sorted out #6  👍

**Comment by NiklasRosenstein** [comment]
Hey @testworksau ,

Sorry for the long silence. I've published 0.5.0 which bumps it's PyJWT dependency to 2.0.0. To be honest I haven't really had the time to test this; could you report back if anything is broken? Thanks!

**Comment by testworksau** [comment]
Hi @NiklasRosenstein - I raised an issue to cover the problem I found during testing:
https://github.com/NiklasRosenstein/python-github-bot-api/issues/7

I've also raised a PR to fix it:
https://github.com/NiklasRosenstein/python-github-bot-api/pull/8
